### PR TITLE
Fix #365: Unable to retry presence check init

### LINKED ISSUE #365: Unable to retry presence check init
When first attempt to init presence check fails, it is not possible to process the second call.
```
2022-09-09 15:35:32.232 ERROR 937272 --- [o-8080-exec-220] c.w.a.o.statemachine.StateMachineConfig  : Not accepted event PRESENCE_CHECK_INIT
2022-09-09 15:35:32.232  WARN 937272 --- [o-8080-exec-220] w.a.o.c.a.IdentityVerificationController : Missing one of important values to generate response entity, response=null, status=null
2022-09-09 15:35:32.418  INFO 937272 --- [o-8080-exec-217] i.g.s.p.a.s.s.v3.PowerAuthServiceImpl    : GetActivationStatusRequest received, activation ID: 28654fe4-4c0c
```

### PR DESCRIPTION
**Pull request by banterCZ**
- Allow to send event PRESENCE_CHECK_INIT during PRESENCE_CHECK_IN_PROGRESS
- Remove IProovConst
- Add another initPresenceCheck to repeatPresenceCheckStartTest
- Improve logging of IdentityVerification phase changes

### CODE REVIEW

File: enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/IdentityVerificationLimitService.java [hunk 1]
```diff
@@ -130,6 +132,7 @@ public void checkDocumentUploadLimit(OwnerId ownerId, IdentityVerificationEntity
             identityVerification.setTimestampLastUpdated(ownerId.getTimestamp());
             identityVerification.setTimestampFailed(ownerId.getTimestamp());
             identityVerificationRepository.save(identityVerification);
+            logger.info("Switched to {}/FAILED; process ID: {}", identityVerification.getPhase(), identityVerification.getProcessId());
```
Comment (banterCZ) [inline enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/IdentityVerificationLimitService.java:135]: Without extensive logging, it was difficult to debug this issue.

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/IProovConst.java [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
- * Constants for iProov purposes
- *
- * @author Lukas Lukovsky, lukas.lukovsky@wultra.com
- */
-public class IProovConst {
-
-    /**
-     * Session parameter name of the verification token
-     */
-    public static final String VERIFICATION_TOKEN = "iproov-verification-token";
```
Comment (banterCZ) [inline enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/IProovConst.java:30]: Simplify namespace, there is no need to have this constant _public_.

File: enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/StateMachineConfig.java [hunk 3]
```diff
@@ -317,6 +317,15 @@ private void configurePresenceCheckTransitions(StateMachineTransitionConfigurer<
                 .action(presenceCheckInitAction)
                 .target(OnboardingState.PRESENCE_CHECK_IN_PROGRESS)
 
+                .and()
+                .withExternal()
+                .source(OnboardingState.PRESENCE_CHECK_IN_PROGRESS)
```
Comment (banterCZ) [inline enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/StateMachineConfig.java:322]: The main change! Allow to send `PRESENCE_CHECK_INIT` event during `PRESENCE_CHECK_IN_PROGRESS`, respectively to call `/api/identity/presence-check/init` to get iProov token. cc @kober32

File: enrollment-server-onboarding/src/test/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProviderTest.java [hunk 4]
```diff
@@ -100,20 +101,22 @@ void getResultTest() throws Exception {
...
-        SessionInfo sessionInfo1 = provider.startPresenceCheck(ownerId);
+        final SessionInfo sessionInfo1 = provider.startPresenceCheck(ownerId);
         assertNotNull(
-                sessionInfo1.getSessionAttributes().get(IProovConst.VERIFICATION_TOKEN),
+                sessionInfo1.getSessionAttributes().get(VERIFICATION_TOKEN),
                 "Missing presence check verification token in session 1"
         );
 
-        SessionInfo sessionInfo2 = provider.startPresenceCheck(ownerId);
+        initPresenceCheck(ownerId);
```
Comment (banterCZ) [inline enrollment-server-onboarding/src/test/java/com/wultra/app/onboardingserver/presencecheck/iproov/provider/IProovPresenceCheckProviderTest.java:110]: To be sure to call init at iProov twice, but works anyway.

**Review by romanstrobl — APPROVED** [review]
Ok, let's deploy this change and test it.

**Pull request merged**